### PR TITLE
Plugins: Move Xpad and USBqemu plugins to old plugins solution

### LINKED ISSUE #106: USBqemu: no device is listed
<b>SPECS</b>
O.S.: Windows 7 x64 Home Premium SP1 with x86 & x64 Visual Studio 2005, 2008, 2010, 2012, 2013.
BUILD: from PCSX2 1.2.1 Stable to PCSX2 1.3.0 (GIT Weekly Builds: 213-g0616708]
PLUGIN: USBqemu 0.1.0 by @gigaherz 

<b>REPRODUCE BUG</b>
1) Run PCSX2.exe and Go to menu: Config > Plugin BIOS Selector.
2) Go to Plugins > USB > USBqemu 0.1.0 and hit APPLY.
3) Go to USBqemu 0.1.0 and hit CONFIGURE.

<b>BUG REPORT</b>
At USBConfigure screen there is no way to select any device at port 1 or 2. The dropdown menu is blank.

usbqemu

If you check "Enable Logging" option or leave it blank, it saves correctly to USBqemu.ini at INIS folder.

<i>USBqemu.ini</i>

<pre>
[Interface]
Logging=FALSE
</pre>

<b>WORKAROUND</b>
1) Exit PCSX2.
2) Backup your old USBqemu.dll
3) Download and extract <a href="http://forum.coldbird.net/viewtopic.php?f=32&amp;t=1606" target="_blank">USBqemu 0.1.0 custom, version 2014/06/08 - by jackun (based on gigaherz's USBqemu)</a> into PLUGINS folder. <a href="https://dl.dropboxusercontent.com/u/235773/USBqemu.dll" target="_blank">[Direct link here]</a>

4) Run PCSX2 again and go to Config > Plugin BIOS Selector > USB > USBqemu 0.1.0 > CONFIGURE.</li>
5) As you can see below, there is a list of devices that you can select.</li>

usbqemu_air_gamer
6) In my case, I selected Keyboard because I wanted to use it with Network Adapter Start-up Disc and Biohazard Outbreak on private server (obsrv.org). Hit OK when you are done.
7) As this custom plugin doesn't work properly while in games (3 and M keys keep pressing until forever), close PCSX2. 
8) Now delete this custom USBqemu plugin from PLUGINS folder. This plugin is useful to create a INI file with the devices that was set early in step 5.

<i>jackun's USBqemu.ini</i>

<pre>
[Interface]
Logging=FALSE
Dev0=1
Dev1=0
Path=
</pre>

9) Copy the original USBqemu backup again into PLUGINS folder.
10) Now you are good to run your game and start chatting online.

If you check again the USBqemu's USBConfigure screen, you still can't list or choose a device. You only can do that with the custom build by jackun's (Air_gamer post at Prometheus).

Bug originally reported at: http://forums.pcsx2.net/Thread-Bug-Report-gigaherz-s-USBqemu-plugin-no-device-is-listed

### LINKED ISSUE #2293: Removing XPad
***PCSX2 version:***
Up to the latest PCSX2 build.

Another user who used the XPad plugin and had controller issues popped up on the forum:
https://forums.pcsx2.net/Thread-RE-Need-for-Speed-Underground-2-SLES-52725-E

The XPad plugin is still included with the latest v1.5.0 builds, even though the LilyPad plugin offers the same functionality(out of the box XInput controls) as XPad without these issues. So I suggest removing this plugin from being included in PCSX2 builds.

This was discussed before( https://github.com/PCSX2/pcsx2/pull/1699#issuecomment-269562266 ), but nothing has come of it yet.

### PR DESCRIPTION
**Pull request by FlatOutPS2**
The XPad plugin has been superseded by the LilyPad plugin, and is known to cause control issues.

The USBqemu plugin is a dead plugin with no functionality.

Let me know if more changes are required. 😛 

Close #106
Close #2293

### CODE REVIEW

**Comment by lightningterror** [comment]
That was fast.

File: old_plugins.sln [hunk 1]
```diff
@@ -198,6 +202,46 @@ Global
...
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Devel|x64.ActiveCfg = Release|x64
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Devel|x64.Build.0 = Release|x64
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release AVX2|Win32.ActiveCfg = Release|Win32
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release AVX2|Win32.Build.0 = Release|Win32
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release AVX2|x64.ActiveCfg = Release|x64
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release AVX2|x64.Build.0 = Release|x64
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release SSE4|Win32.ActiveCfg = Release|Win32
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release SSE4|Win32.Build.0 = Release|Win32
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release SSE4|x64.ActiveCfg = Release|x64
+		{6F3C4136-5801-4EBC-AC6E-37DF6FAB150A}.Release SSE4|x64.Build.0 = Release|x64
```
Comment (turtleli) [inline old_plugins.sln]: Legacy GSdx wasn't added to this solution, so there shouldn't be any Release SSE4, Release AVX2 lines.

**Comment by lightningterror** [comment]
I never checked how long it takes for each of them to compile, I wonder how fast will appveyor and travis compile now without them.

**Comment by FlatOutPS2** [comment]
> I never checked how long it takes for each of them to compile, I wonder how fast will appveyor and travis compile now without them.

I doubt there will be much difference. Xpad and USBqemu compile in seconds.

**Comment by MrCK1** [comment]
Could PeopsSPU2 and ZeroSPU2 possibly be added here, or is that out of the scope for this?

**Comment by FlatOutPS2** [comment]
> While we're at it, could PeopsSPU2 and ZeroSPU2 possibly be added here, or is that out of the scope for this?

They've already been moved to oldplugins, so yes that's out of scope here.

**Pull request merged**